apps/evans/components/core/AudioPlayer.tsx

```diff
@@ -11,7 +14,7 @@ interface Props {
   }>;
 }
 
-const AudioPlayer: React.FC<Props> = ({ tracks }) => {
+const AudioPlayer: (props: Props) => React.JSX.Element | null = ({ tracks }) => {
```

Comment:
yeah, i had to do this because this component is used in our mdx files, and i couldn't get it to typecheck without explicitly typing it this way. seems to be a limitation or deficiency of the types for the rsc mdx stuff.  i think the difference with this explicit type is that i've removed some other options (i think `undefined`).